assets/src/tinymce/__tests__/utils/index.test.js

```diff
@@ -61,3 +78,23 @@ describe('Test view is circle', () => {
     expect(circleView).toBeTruthy();
   });
 });
+
+/**
+ * Test shortcode build function.
+ */
+describe('Test shortcode is prepared', () => {
+  it('test without editor instance', () => {
+    select.mockImplementationOnce(() => ({
+      getEditor: () => null,
+      getCurrentViewSettings: () => null,
+    }));
+
+    const shortCode = Utils.prepareShortCode();
+    expect(shortCode).toBe('[testStories /]');
+  });
+
+  it('test with editor instance set', () => {
+    const shortCode = Utils.prepareShortCode();
+    expect(shortCode).toBe('[testStories order=latest dummy=true /]');
```

Comment:
Aren't shortcode attribute values usually in double quotes? Does it cause issues if not? What if the value contains a space?